README.md

```diff
@@ -0,0 +1,33 @@
+# Core Snap
+
+This repository contains the official Core snap. The core snap plays
```

Comment:
Ah, I wanted to write some more and I think I accidentally vim-removed that. I wanted to explain what the core snap is for.